sections/introduction.tex

```diff
@@ -34,7 +34,14 @@ \section{Introduction}\label{sec:introduction}
 	\item \textit{How does including metadata within the \gls{lda} model impact the resulting topics?}
 	\item \textit{How can multiple metadata fields be integrated within a topic model?}
 \end{itemize}
-\todo[inline]{contributions}
+
+Our work is similar to that of \citet{MetaLDA2017}, since we working with the meta information and combining it into an \gls{lda} model.
```

Comment:
```suggestion
Our work is similar to that of \citet{MetaLDA2017} since we work with meta information and applying it into an \gls{lda} model.
```

Reply:
```suggestion
Our work is similar to that of \citet{MetaLDA2017} since we work with meta information and are applying it into an \gls{lda} model.
```
Hvis du vil have applying i stedet for apply.

Reply:
```suggestion
Our work is similar to that of \citet{MetaLDA2017} since we work with meta information and are applying it to an \gls{lda} model.
```